dask/dataframe/io/tests/test_csv.py

```diff
@@ -730,8 +731,8 @@ def test_read_csv_sep():
     charlie###300""")
 
     with filetext(sep_text) as fn:
-        ddf = dd.read_csv(fn, sep="###")
-        df = pd.read_csv(fn, sep="###")
+        ddf = dd.read_csv(fn, sep="###", engine="python")
```

Comment:
Yes, this warning comes from pandas (the C csv engine doesn't allow multi-character separators, so it falls back to the python engine with a warning)